backend/src/data/migrations/20220816134616_add_unique_relation_constraint.ts

```diff
@@ -0,0 +1,37 @@
+import { Knex } from 'knex';
+
+const TableName = {
+  GROUPS_TO_PERMISSIONS: 'groups_to_permissions',
+  USERS_TO_GROUPS: 'users_to_groups',
+  PERMISSIONS: 'permissions',
+  GROUPS: 'groups',
+  USERS: 'users',
```

Comment:
Seems like you're using only these two values
```suggestion
  GROUPS_TO_PERMISSIONS: 'groups_to_permissions',
  USERS_TO_GROUPS: 'users_to_groups',
```